stable/hubot/values.yaml

```diff
@@ -0,0 +1,61 @@
+# Default values for hubot.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+namespace: default
```

Comment:
Remove namespace from `values.yaml`. You pass the namespace to the Helm command on installation.